### DISCUSSION TITLE
ufe and maya version relationship

### DISCUSSION THREAD

**Discussion by wandth** [Autodesk/maya-usd#2594] [Q&A] [answered]
I found some test is require ufe 0.4.1
such as

> https://github.com/Autodesk/maya-usd/blob/dd195b494e31fc2580765051d4f2fd5e64a501a5/test/lib/ufe/testShaderNodeDef.py#L54

is that require Maya 2024?

colud someone tell me the Maya UFE version and Maya version relationship

**Comment by seando-adsk** [comment, marked as answer]
@wandth Ufe is evolving all the time and there have been major updates with each major update of Maya. Please see [build.md](https://github.com/Autodesk/maya-usd/blob/dev/doc/build.md) and in the "Universal Front End (UFE)" section it lists what version of Ufe was used for each Maya version.

**Reply by seando-adsk** [reply]
@wandth FYI, I updated the build.md with a table for the Ufe version used in Maya version to help make this easier to understand.
